elastic/kibana
# [Guided onboarding] Fix continue button behavior

Fixes https://github.com/elastic/kibana/issues/143306

Jest tests were also updated as part of this PR to verify the dropdown panel behaves as expected.

### How to test
Please verify the step button works as expected for "Start", "Continue" and manual completion/"Mark done".
